src/pageContent/Profile/index.tsx

```diff
@@ -0,0 +1,78 @@
+import Text from "src/components/Text";
+import Button from "src/components/Button";
+import Input from "src/components/Input";
+import { useState } from "react";
+
+const ProfileContent: React.FC<Record<string, never>> = () => {
+  const [name, setName] = useState<string>(undefined);
+  const [subject, setSubject] = useState<string>(undefined);
+  const [content, setContent] = useState<string>(undefined);
+
+  const handleSubmit = (event: React.MouseEvent) => {
+    event.preventDefault();
+
+    if (!name) {
+      setName("");
+      return;
+    }
+    if (!subject) {
+      setSubject("");
+      return;
+    }
+    if (!content) {
+      setContent("");
+      return;
+    }
+
+    const body =
+      `Hello,\n` +
+      `${content}\n\n` +
+      `Thank you.\n\n` +
+      `Yours sincerely,\n` +
+      `${name}\n`;
+
+    location.href =
+      `mailto:alissayarmantho1@gmail.com` +
+      `?subject=${encodeURIComponent(subject)}` +
+      `&body=${encodeURIComponent(body)}`;
+  };
+
+  return (
+    <>
+      <div className="grid grid-cols-12 grid-rows-8">
```

Comment:
as much as possible actually use flex instead of grid, if you use grid, the mobile version is broken pretty bad

![image](https://user-images.githubusercontent.com/51783522/131252433-9257dc98-bb75-4264-ae42-a4cf4ce8c60b.png)

Reply:
Ok